var/spack/repos/builtin/packages/minizip/package.py

```diff
@@ -21,3 +21,21 @@ class Minizip(AutotoolsPackage):
     depends_on('libtool', type='build')
     depends_on('m4', type='build')
     depends_on('zlib')
+
+    # build minizip and miniunz
+    @run_before('autoreconf')
+    def build_minizip_binary(self):
+        print(self)
```

Comment:
You can remove print statements

Reply:
updated

Reply:
I think you forgot to push the commit to remove this print statement.

Reply:
Ugh, I made a mistake and remove the comment
`# build minizip and miniunz` and `# install minizip and miniunz`